lib/create-plugin.js

```diff
@@ -10,6 +10,16 @@ const errorHelper = new ErrorHelper
 function createPlugin(instrumentationApi) {
   return {
     requestDidStart() {
+      const requestParent = instrumentationApi.getActiveSegment()
```

Comment:
Grabbing and passing this in is technically not requires (createSegment will do that) but I like the explicitness and we may find we want it for other uses anyways.